docker-examples/docker-compose-postgres.yaml

```diff
@@ -28,9 +28,9 @@ services:
       - RUST_BACKTRACE=1
       - RUST_LOG=info,h2=warn,hyper=warn,tower=warn
       - GOLEM__REDIS__HOST=redis
-      - GOLEM__REDIS__PORT=${REDIS_PORT}
-      - GOLEM__HTTP_PORT=${SHARD_MANAGER_HTTP_PORT}
-      - GOLEM_SHARD_MANAGER_PORT=${SHARD_MANAGER_GRPC_PORT}
+      - GOLEM__REDIS__PORT=6379
```

Comment:
So now it closes the opportunity to use a single .env file that has all configurations in 1 place. Hardcoded redis port!